From 23a42d308fd21d5dae0d519b1a63f099df1eaa4c Mon Sep 17 00:00:00 2001
From: Omar Ramirez Luna <omar.r.ramirez.luna@intel.com>
Date: Wed, 12 Oct 2016 17:32:03 -0700
Subject: [PATCH] intel-aero: shutdown to handle LED toggling

During shutdown we need to indicate to user that the button press
was registered, as soon as the event is handled we flash
the LED_BLUE_IO on/off for 1 second every 100 ms.

Signed-off-by: Omar Ramirez Luna <omar.r.ramirez.luna@intel.com>
---
 actions/power.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/actions/power.sh b/actions/power.sh
index 8b76ef5..09fe10b 100644
--- a/actions/power.sh
+++ b/actions/power.sh
@@ -1,3 +1,4 @@
 #!/bin/sh
 
+/usr/bin/led_ctrl shutdown_seq
 /sbin/shutdown -h now "Power button pressed"
-- 
1.9.1

